docs/src/styled.d.ts

```diff
@@ -2,10 +2,5 @@ import '@emotion/react';
 import type {DoobooTheme} from 'dooboo-ui';
 
 declare module '@emotion/react' {
-  export interface Theme extends DoobooTheme {
-    isPortrait?: boolean;
-    isMobile?: boolean;
-    isTablet?: boolean;
-    isDesktop?: boolean;
-  }
+  export interface Theme extends DoobooTheme {}
```

Comment:
Previously, one of our `eslint` rule avoid us writing empty interface `{}` so we had it.

I am not sure why it doesn't complain anymore.

Reply:
@hyochan  Thanks for sharing background information!